static/app/components/codecov/datePicker/dateSelector.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import {ConfigFixture} from 'sentry-fixture/config';
+import {UserFixture} from 'sentry-fixture/user';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {DateSelector} from 'sentry/components/codecov/datePicker/dateSelector';
+import ConfigStore from 'sentry/stores/configStore';
+
+const {router} = initializeOrg({
+  router: {
+    location: {
+      pathname: '/codecov/tests/',
+      query: {},
+    },
+    params: {},
+  },
+});
+
+describe('DateSelector', function () {
+  const onChange = jest.fn();
+
+  function getComponent(props = {}) {
+    return <DateSelector onChange={onChange} {...props} />;
+  }
+
+  function renderComponent(props = {}) {
+    return render(getComponent(props), {router});
+  }
+
+  beforeEach(function () {
+    ConfigStore.loadInitialData(
+      ConfigFixture({
+        user: UserFixture({
+          options: {...UserFixture().options, timezone: 'America/New_York'},
+        }),
+      })
+    );
```

Comment:
this can be simplified a bit
```suggestion
    ConfigStore.set('user', UserFixture({timezone: 'America/New_York'}));
```

Reply:
you probably don't need to setup the config store at all